lua/gitportal/config.lua

```diff
@@ -15,6 +16,30 @@ M.options = default
 function M.setup(options)
   -- Merge user options with default options
   M.options = vim.tbl_deep_extend("force", {}, default, options or {})
+
+  local commands = {
+    open = function ()
+      require("gitportal").open_file_in_browser()
+    end,
+    link = function ()
+      require("gitportal").open_file_in_neovim()
+    end
```

Comment:
I chose these at random, but maybe we can come up with something more obvious

Reply:
I'm thinking a lot about naming, since these are names we won't really want to change once they are available. 

I think we should be very descriptive with the names. Perhaps they should have a prefix like `GP`?

Maybe something like 

```lua
GPBrowse -- Opens current file in browser
GPOpen or GPOpenLink -- Accepts a permalink and opens it in neovim


-- And later down the road... (just spit ballin' here)
GPCopy -- Generates a permalink for the current file you're in and copies it to clipboard
GPPullRequest -- Generates permalink for pull request
GPBrowsePR -- Open the link for the pull request of the current branch in browser
```

What do you think?

Reply:
@tronikelis 
> This is a known bad practice, you don't want to pollute the user command space, ideally it should be one user command "GitPortal [open/list/etc..]" with arguments which action to execute

That is very helpful to know, and that makes sense. I see that's how you have it set up here. I wasn't really understanding off the bat that the commands are specified as argument to the user command `:GitPortal`

I think for opening files in the browser, I think `:GitPortal browse` would be a good name for such command (I also like leaving that as the default command if a user calls `GitPortal`)

I'm struggling with the name for opening a file locally in neovim with a permalink. I think `link` is okay, I would prefer something more explicit like `openLink` but that is an awful lot of characters to type (`:GitPortal openLink`). Presumably, the user will bind the functions/commands they are interested in to some keys and will not be typing their most used commands regularly. 

What are your thoughts? We could always abbreviate the `:GitPortal` command to `:GP`, tho that increases the risk for clashing with other plugins that had the same idea (none come to mind as of now).

Reply:
hmm, I don't really have an opinion on this, feel free to edit the PR when you have decided on the naming

Reply:
Okay, I'll give it some thought